test/e2e/framework/test_context.go

```diff
@@ -153,9 +153,6 @@ type TestContextType struct {
 	// Node e2e specific test context
 	NodeTestContextType
 
-	// Indicates what path the kubernetes-anywhere is installed on
-	KubernetesAnywherePath string
```

Comment:
TestContextType does not have publicly exposed support guarantees, yet.

Reply:
@alejandrox1 😬